src/manager.js

```diff
@@ -305,50 +320,99 @@ AdManager.prototype.setSlotTargeting = function(element, slot) {
  * @returns {Element} - Fully configured ad slot
 */
 AdManager.prototype.configureAd = function (element) {
+
+  var slotName = this.options.dfpSiteCode || "",
+    kinjaMeta = window.kinja.meta,
+    postMeta = window.kinja.postMeta || {},
+    currentBlog = kinjaMeta.blog,
+    blogSales = kinjaMeta.blogSales,
+    pageType = kinjaMeta.pageType,
+    getExperimentVariation = function(scope) {
```

Comment:
I think it'd be good to move these function declarations (`getExperimentVariation`, `getExperimentId`, and `getQueryParameter` out of being defined within this function closure, and defined as separate functions that we can easily unit test.  It might even make sense to break them out into complete separate utility files/helpers that have their own spec file